backend/src/server/routes/v3/dashboard-router.ts

```diff
@@ -0,0 +1,610 @@
+import { z } from "zod";
```

Comment:
I think the total count breaks for replicated imports? 
![image](https://github.com/user-attachments/assets/afa58d28-b430-45fd-84e3-b881054936ae)

As you can see I only have 3 import entries but the total count is showing as 5 (at the bottom left)

Reply:
excluding replicas in count now